lib/validators/movie.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).required(),
```

Comment:
we can take out `.min(1)`, since empty strings are disallowed by default. 
[see here](https://joi.dev/api/?v=17.2.1#string)